from app.exceptions import AppError


class DrawingError(AppError):
    pass
